flutter_web_auth_2/ios/Classes/SwiftFlutterWebAuth2Plugin.swift

```diff
@@ -64,7 +66,11 @@ public class SwiftFlutterWebAuth2Plugin: NSObject, FlutterPlugin {
             if #available(iOS 12, *) {
                 var _session: ASWebAuthenticationSession? = nil
                 if #available(iOS 17.4, *) {
-                    _session = ASWebAuthenticationSession(url: url, callback: ASWebAuthenticationSession.Callback.customScheme(callbackURLScheme), completionHandler: completionHandler!)
+                    if (callbackURLScheme == "https") {
+                        _session = ASWebAuthenticationSession(url: url, callback: ASWebAuthenticationSession.Callback.https(host: host, path: path), completionHandler: completionHandler!)
```

Comment:
Could you please add an `if` check here which throws a "good"/"readable" error message when `callbackURLScheme == "https"`, but `host` is `null` or `path` is `null`?

Reply:
Hi, thanks for the quick feedback! I made the adjustments you requested and it seems to work as expected.